### PR TITLE
change to use serif instead of sans for the heading and better align the follow button

### PR DESCRIPTION
**Pull request by bjfletcher**
Go from:
image

to:

image

Waiting for confirmation from UX that that's what they want.

/cc @andygnewman @i-like-robots

### CODE REVIEW

File: client/components/sub-header/_main.scss [hunk 1]
```diff
@@ -176,8 +176,8 @@
 }
 
 .sub-header__page-title {
-	@include oTypographySansData(xl, $load-progressively: true);
-	@include oTypographyFallbackFontSize(sansData, 40px);
+	@include oTypographySerif(xl, $load-progressively: true);
```
Comment (andygnewman) [inline client/components/sub-header/_main.scss]: I think this should be SerifDisplay https://github.com/Financial-Times/o-typography#serif

File: client/components/sub-header/_main.scss [hunk 2]
```diff
@@ -176,8 +176,8 @@
 }
 
 .sub-header__page-title {
-	@include oTypographySansData(xl, $load-progressively: true);
-	@include oTypographyFallbackFontSize(sansData, 40px);
+	@include oTypographySerifDisplay(xl, $load-progressively: true);
+	@include oTypographyFallbackFontSize(serifDisplay, 40px);
```
Comment (bjfletcher) [inline client/components/sub-header/_main.scss]: @andygnewman I've changed to SerifDisplay. I'm not sure the fallback font will work. Do you know?

Comment (andygnewman) [inline client/components/sub-header/_main.scss]: Should do https://github.com/Financial-Times/o-typography/blob/master/scss/_progressive-font.scss#L42

Comment (bjfletcher) [inline client/components/sub-header/_main.scss]: Great, thanks Andy :)

**Pull request merged**